packages/@okta/vuepress-theme-prose/components/AssuranceItem.vue

```diff
@@ -0,0 +1,24 @@
+<template>     
+    <div class="assurance-item">
+        <div class="assurance-item--rounded"> 
+            <img :src="item.img"/>
+        </div>
+        <div class="assurance-item--content">
+            <div class="assurance-item--content--title">
+              {{item.title}}
+            </div>
+            <div class="assurance-item--content--description">
+              {{item.description}}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+  export default {
+    name: 'AssuranceItem',
+    props: ['item'],
```

Comment:
I think it would be nice to finish the props declaration to describe the properties `title`, `description`, and `img`